src/licensedcode/data/licenses/mit-x11.LICENSE

```diff
@@ -0,0 +1,9 @@
+Copyright (C) 1996 X Consortium
+
```

Comment:
Are you sure this is not the same as the existing `x11-xconsortium.LICENSE`  ? if not the same what would be the differences?

Reply:
Oh sorry my bad! Actually I did a `git grep mit-x11` and since no result was found I added it. I will delete these files and add the licenses missing from clipol.org in this PR.